tests/superphot_plus/test_lightcurve.py

```diff
@@ -32,6 +32,33 @@ def test_create(tmp_path):
         lc2 = Lightcurve(times2, fluxes, bands, errors)
 
 
+def test_max_flux():
+    times = np.array(range(11))
+    fluxes = np.array([1.1, 0.2, 0.3, 0.1, 100.2, 20.3, 0.1, 0.1, 1.1, 0.1, 1000.0])
+    bands = np.array(["r", "r", "r", "r", "g", "r", "r", "r", "g", "g", "b"])
+    errors = np.array([0.1] * 11)
+    lc = Lightcurve(times, fluxes, errors, bands)
+
+    # Test all bands with a standard adjustment (-1.0 * |error|)
+    all_max, all_max_t = lc.find_max_flux()
+    assert pytest.approx(999.9) == all_max
```

Comment:
The yoda asserts are throwing me off. Is this pythonic?

Reply:
I'm not sure what you mean, but I'm guessing it is the value == variable format. I changed it throughout the test.